main/main.cpp

```diff
@@ -1621,9 +1553,9 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 	/* Initialize Pen Tablet Driver */
 
 	{
-		GLOBAL_DEF_RST_NOVAL("input_devices/pen_tablet/driver", "");
-		GLOBAL_DEF_RST_NOVAL("input_devices/pen_tablet/driver.windows", "");
-		ProjectSettings::get_singleton()->set_custom_property_info("input_devices/pen_tablet/driver.windows", PropertyInfo(Variant::STRING, "input_devices/pen_tablet/driver.windows", PROPERTY_HINT_ENUM, "wintab,winink"));
+		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "input_devices/pen_tablet/driver"), "");
+		GLOBAL_DEF_RST_NOVAL(PropertyInfo(Variant::STRING, "input_devices/pen_tablet/driver.windows"), "");
+		ProjectSettings::get_singleton()->set_custom_property_info(PropertyInfo(Variant::STRING, "input_devices/pen_tablet/driver.windows", PROPERTY_HINT_ENUM, "wintab,winink"));
```

Comment:
Why isn't it combined as well?

Reply:
I made `GLOBAL_DEF` ignore PropertyInfo for overrides so that it will automatically use the original property's PropertyInfo, and we don't have to duplicate it in the code (see project_settings.cpp line 1051). This is the only setting where the override property needs a different PropertyInfo than it's original property.